dev-cpp/yaml_cpp/yaml_cpp5-0.5.3.recipe

```diff
@@ -4,30 +4,36 @@ DESCRIPTION="yaml-cpp is a YAML parser and emitter in C++ matching the YAML \
 HOMEPAGE="https://github.com/jbeder/yaml-cpp/"
 COPYRIGHT="2008 Jesse Beder"
 LICENSE="MIT"
-REVISION="2"
+REVISION="3"
```

Comment:
Since you're renaming the package already, revision shouldn't be bumped

```diff
-REVISION="3"
+REVISION="1"

Reply:
@alaviss: Yeah, we _could_ reset `REVISION` to `1`, but I think it is better to increment it, like @extrowerk did.

(I I have to admit, however, that I don't know if there is a rule about this. All I know is that I would choose to a revbump.)

Reply:
Renaming implies resetting the revision (that is, if no yaml_cpp5-0.5.3 package ever existed).

Reply:
Okay
(I thought it would be fine to have an exception here, but, after all, let's avoid uneeded exceptions :)

Reply:
@alaviss, we reached a concensus :tada: :smile:
(I'm adding a :+1: to your comment about resetting `REVISION`.)